### ISSUE TITLE
Tag class name as style

### ISSUE THREAD

**Issue by rosostolato** [Aymkdn/html-to-pdfmake#11]
I have the following html string with class names:

```
<span class="fr-img-caption fr-fic fr-dib" style="width: 166px;">
  <span class="fr-img-wrap">
    <img src="...base64string">
    <span class="fr-inner">Image tag</span>
  </span>
</span>
```

output:

```
[
  {
    "stack": [
      {
        "image": "...base64string",
        "style": [
          "html-img"
        ]
      },
      {
        "text": "Image tag",
        "style": [
          "fr-inner", // span class
          "html-span"
        ]
      }
    ],
    "style": [
      "html-span",
      "html-span"
      // class was not inserted here
    ]
  }
]
```

The last span tag has 'fr-inner' inside his style array, but the parent ones do not. Is it possible to insert these class names inside their style array?

**Comment by Aymkdn** [comment]
I've just left on vacation for two weeks. I'll have a look at it on my return.

**Comment by rosostolato** [comment]
Created a PR that make it happen #12

**Comment by Aymkdn** [comment]
That should be resolved with v1.0.7
